src/containers/upload-page/store/sagas.js

```diff
@@ -1031,32 +1033,41 @@ function* uploadTracksAsync(action) {
       )
       return
     }
-    const newEndpoint = selectedServices.join(',')
+    newEndpoint = selectedServices.join(',')
+  }
 
-    // Try to upgrade to creator, early return if failure
-    try {
-      console.debug(`Attempting to upgrade user ${user.user_id} to creator`)
+  // Try to upgrade to creator, early return if failure
+  let metadata = { is_creator: true }
+  try {
+    console.error(`Attempting to upgrade user ${user.user_id} to creator`)
+    if (newEndpoint && newEndpoint.length > 0) {
+      // If new endpoint was selected via the selection logic above, update
+      // the user's creator_node_endpoint and is_creator flag
+      console.error(`Upgrading user ${user.user_id} to creator...`)
```

Comment:
console.error?

Reply:
ah, I wanted the console logs to stick out when I was viewing them in the browser. will update. is console logging ok or should we not log anything?